examples/interactions/param.py

```diff
@@ -63,7 +64,7 @@ async def full_description(
 @bot.slash_command()
 async def defaults(
     inter: disnake.CommandInteraction,
-    string: str = None,
+    string: Optional[str] = None,
```

Comment:
can we possibly ignore this somehow because i think the example proves its point better *without* Optional

Reply:
https://github.com/DisnakeDev/disnake/pull/751/commits/cf042d31f7035a28e1b7de37ef18fb358c68140d